dagmc_stats/dagmc_stats.py

```diff
@@ -199,7 +199,7 @@ def get_tri_side_length(my_core, tri):
     side_lengths : a list of side lengths of triangle
```

Comment:
Docstring needs updating because `side_lengths` is now a dictionary. This should describe the dict structure.